group_variables.go

```diff
@@ -42,6 +42,7 @@ type GroupVariable struct {
 	Protected        bool              `json:"protected"`
 	Masked           bool              `json:"masked"`
 	EnvironmentScope string            `json:"environment_scope"`
+	Raw              bool              `json:"raw"`
```

Comment:
Following the ordering from the docs... So can you please place `Raw` in between `Masked` and `EnvironmentScope` everywhere?